changelog/Microsoft.IdentityGovernance.AccessReviews.json

```
@@ -663,6 +663,80 @@
+            },
+            {
+               "Id":"da428e60-690f-11ec-ae3f-e486bb3b9c86",
+               "ApiChange":"Relationship",
+               "ChangedApiName":"stages",
+               "ChangeType":"Addition",
+               "Description":"Added the **stages** relationship to the [accessReviewInstance](https://docs.microsoft.com/en-us/graph/api/resources/accessReviewInstance?view=graph-rest-beta) resource.",
+               "Target":"accessReviewInstance"
+            },
+            {
+               "Id":"da44f0e7-690f-11ec-ae3f-e486bb3b9c86",
+               "ApiChange":"Resource",
+               "ChangedApiName":"accessReviewStageSettings",
+               "ChangeType":"Addition",
+               "Description":"Added the [accessReviewStageSettings](https://docs.microsoft.com/en-us/graph/api/resources/accessReviewStageSettings?view=graph-rest-beta) resource type.",
+               "Target":"accessReviewStageSettings"
+            },
+            {
+               "Id":"da4754e2-690f-11ec-ae3f-e486bb3b9c86",
+               "ApiChange":"Resource",
+               "ChangedApiName":"accessReviewStage",
+               "ChangeType":"Addition",
+               "Description":"Added the [accessReviewStage](https://docs.microsoft.com/en-us/graph/api/resources/accessReviewStage?view=graph-rest-beta) resource type.",
+               "Target":"accessReviewStage"
+            },
+            {
+               "Id":"da49b6c4-690f-11ec-ae3f-e486bb3b9c86",
+               "ApiChange":"Enum type",
+               "ChangedApiName":"accessReviewStageFilterByCurrentUserOptions",
+               "ChangeType":"Addition",
+               "Description":"Added the **accessReviewStageFilterByCurrentUserOptions** enumeration type.",
+               "Target":"accessReviewStageFilterByCurrentUserOptions"
+            },
+            {
+               "Id":"da4c17a4-690f-11ec-ae3f-e486bb3b9c86",
+               "ApiChange":"Method",
+               "ChangedApiName":"stop",
+               "ChangeType":"Addition",
+               "Description":"Added the [stop](https://docs.microsoft.com/en-us/graph/api/accessReviewStage-stop?view=graph-rest-beta) method to the [accessReviewStage](https://docs.microsoft.com/en-us/graph/api/resources/accessReviewStage?view=graph-rest-beta) resource.",
+               "Target":"accessReviewStage"
+            },
+            {
+               "Id":"da4e7b49-690f-11ec-ae3f-e486bb3b9c86",
+               "ApiChange":"Method",
+               "ChangedApiName":"filterByCurrentUser",
+               "ChangeType":"Addition",
+               "Description":"Added the [filterByCurrentUser](https://docs.microsoft.com/en-us/graph/api/accessReviewStage-filterByCurrentUser?view=graph-rest-beta) method to the [accessReviewStage](https://docs.microsoft.com/en-us/graph/api/resources/accessReviewStage?view=graph-rest-beta) resource.",
+               "Target":"accessReviewStage"
+            }
```

These are already catered for by the "and associated methods" statement in the entry for accessReviewStage.

Instead, I recommend that you add entries for the new PATCH, GET, and filterByCurrentUser methods that you've introduced in accessReviewInstanceDecisionItem docs. Unlike the existing methods, these methods act on stages of an accessReviewInstance rather than just the accessReviewInstance.
```suggestion
            }
```